images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,181 @@
+get_redis_value() {
+	local key="$1"
+	printf 'GET %s\r\n' "$key" | nc -w 5 redis 6379 | awk 'NR==2 {gsub(/\r/, ""); print}'
+}
+
+function openvpn_preconfig {
+	mkdir -p /dev/net
+	if [ ! -c /dev/net/tun ]; then
+		mknod /dev/net/tun c 10 200
+	fi
+	ip -6 route show default 2>/dev/null
+	if [ $? = 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)